civihr_employee_portal/src/View/StaffDirectoryView.php

```diff
@@ -16,13 +16,19 @@ public function alter($view, $query) {
     $queryParts = [];
     $this->getWhereFields($query->where, $queryParts);
     $departmentField = 'hrjc_role_hrjc_revision.role_department';
+    $titleField = 'hrjc_role_hrjc_revision.role_title';
     $roleEndField = 'hrjc_role_hrjc_revision.role_end_date';
+    $roleStartField = 'hrjc_role_hrjc_revision.role_start_date';
 
-    // if department is set we must limit to only current roles
-    if (!empty($queryParts[$departmentField])) {
-      $group = $query->set_where_group('OR'); // add a new WHERE group
-      $query->add_where($group, $roleEndField, date('Y-m-d'), '>=');
-      $query->add_where($group, $roleEndField, NULL, 'IS');
+    // if department or title is set we must limit to only current roles
+    if (!empty($queryParts[$departmentField]) ||
```

Comment:
Doesn't this fit in the 80 character limit on one line?

Reply:
Will check and update as appropriate 👓